### PR TITLE
Trying to hook up trigger, not quite successful

### PR DESCRIPTION
**Pull request by jiwalker-usgs**
@jread-usgs added a bit to figure code, hoping we don't collide

### CODE REVIEW

File: scripts/3_visualize/visualizeLandUse.R [hunk 1]
```diff
@@ -124,7 +124,7 @@ injectLabelTextBreaks <- function(svg.side){
 
 createBarFig <- function(gs.conc, gs.landuse, target_name){
   gs.landuse$global$par$mar <- c(9.1, 4.1, 13.5, 2.1)
-  svg <- dinosvg::svg(gs.landuse, width = 6, height = 6.3, as.xml=TRUE)
+  svg <- dinosvg::svg(gs.landuse, width = 6, height = 6.3, as.xml=TRUE, onload="init(evt)")
```
Comment (jordansread) [inline scripts/3_visualize/visualizeLandUse.R:200]: looks good to me

**Comment by jiwalker-usgs** [comment]
@jread-usgs I resolved conflict, does this look alright?

**Comment by jordansread** [comment]
👍

**Comment by ldecicco-USGS** [comment]
Should the sort be triggered now? I still don't see it happening.

**Comment by jiwalker-usgs** [comment]
@ldecicco-USGS not yet, this sends the trigger event, but it still needs hooking up

**Pull request merged**